scripts/bootstrap.sh

```diff
@@ -111,6 +108,10 @@ fi
 if $installing; then echo "$(date -I) -- installed $version" >> /root/events.log; fi
 if $upgrading; then echo "$(date -I) -- upgraded to $version" >> /root/events.log; fi
 
+if [ "$master_ipaddr" = "masterless" ]; then
+    # ignore given IP parameter and use the one we can detect
+    master_ipaddr=$(ifconfig eth0 | awk '/inet / { print $2 }' | sed 's/addr://')
```

Comment:
erm. I don't know. lost to history? I'll see if `127.0.0.1` works and if so, will pass that around instead of `masterless`.